firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/core/CrashlyticsController.java

```diff
@@ -600,6 +623,11 @@ public ReportUploader createReportUploader(AppSettingsData appSettingsData) {
     };
   }
 
+  private SessionReportingCoordinator.SendReportPredicate shouldSendViaDataTransport(
+      int reportUploadVariant) {
+    return () -> REPORT_UPLOAD_VARIANT_DATATRANSPORT == reportUploadVariant;
```

Comment:
This turned out really nice. 


Should this be a static method?

Reply:
Yes: https://github.com/firebase/firebase-android-sdk/pull/1296#discussion_r385919219